frontend/src/components/common/Filter/components/FilterContent/components/FilterViews/components/FilterView/style.ts

```diff
@@ -11,7 +11,7 @@ interface PanelProps {
   panelWidth: number;
 }
 
-interface ScrollProps {
+interface ScrollProps extends CommonThemeProps {
```

Comment:
Just out of curiosity - what is the purpose of extending `CommonThemeProps`?

Reply:
Great question! Extending CommonThemeProps is basically telling Typescript that we're using CommonThemeProps' props in our styled component, so it won't throw errors incorrectly!